pkg/cache/cache.go

```diff
@@ -86,6 +86,9 @@ type Informer interface {
 	HasSynced() bool
 }
 
+// SelectorsByObject associate a client.Object's GVK to a field/label selector
+type SelectorsByObject map[client.Object]internal.Selectors
```

Comment:
Doesn't the `internal.Selectors` need aliasing as well in order to construct a `SelectorsByObject`?

Reply:
I have create a release-0.8 backport of this PR to be able to consume it at kubernetes-nmstate https://github.com/kubernetes-sigs/controller-runtime/pull/1480, the kubernetes-nmstate PR is working fine https://github.com/nmstate/kubernetes-nmstate/pull/728